tox.ini

```diff
@@ -58,7 +59,17 @@ commands = coverage erase
 skip_install = true
 deps = coverage
 
+[testenv:func_test]
+commands_pre =
+    bash -c ./docker/docker-up-vinyldns.sh
+commands_post =
+    bash -c ./docker/remove-vinyl-containers.sh
+commands =
+    {posargs:pytest -vv func_tests}
+whitelist_externals =
+    bash
+
 [travis]
 python =
   2.7: check, py27
-  3.6: check, py36
+  3.6: check, py36, func_test
```

Comment:
check it, we now spin up docker and run `func_tests` against the docker setup, on every build!  So wholesome